File: Python/Program/Clothing.py
from Product import Product

class Clothing(Product):
    def __init__(self):
        # Memanggil konstruktor kelas induk (Product)
        super().__init__()
        
        # Inisialisasi atribut kelas Clothing
        self.size = ""
        self.material = ""
        self.gender = ""

    def set_size(self, size):
        # Metode untuk mengatur ukuran pakaian
        self.size = size

    def set_gender(self, gender):
        # Metode untuk mengatur jenis kelamin pakaian
        self.gender = gender

    def set_material(self, material):
        # Metode untuk mengatur material pakaian
        self.material = material

    def get_size(self):
        # Metode untuk mendapatkan ukuran pakaian
        return self.size

    def get_material(self):
        # Metode untuk mendapatkan material pakaian
        return self.material

    def get_gender(self):
        # Metode untuk mendapatkan jenis kelamin pakaian
        return self.gender

    def __del__(self):
        pass  # Destruktor untuk membersihkan objek (tidak melakukan apa pun dalam kasus ini)

File: PHP/Program/Clothing.php
<?php

require("Product.php");
class Clothing extends Product{

    // PRIVATE ATRIBUT
    private $size = "";
    private $material = "";
    private $gender = "";

    // CONSTRUCT
    function __construct($size = "", $material = "", $gender = ""){
        $this->size = $size;
        $this->material = $material;
        $this->gender = $gender;
    }

    // PUBLIC METHOD (SETTER & GETTER)
    function setsize($size = ""){
        $this->size = $size;
    }

    function getsize(){
        return $this->size;
    }

    function setmaterial($material = ""){
        $this->material = $material;
    }

    function getmaterial(){
        return $this->material;
    }

    function setgender($gender = ""){
        $this->gender = $gender;
    }

    function getgender(){
        return $this->gender;
    }
    
    // DESTRUCTOR
    function __destruct(){
    }
}

?>